generate/proto.go

```diff
@@ -45,6 +46,14 @@ func protoStringOrNil(s string) *string {
 	return proto.String(s)
 }
 
+func protoEnumValue(value constant.Value) int32 {
+	val, ok := constant.Int64Val(value)
+	if !ok {
+		panic(fmt.Sprintf("value %#v was not an int", value))
```

Comment:
this should now be amended - say "was not exact".

> If x is Unknown, the result is (0, false).

Reply:
Err, I guess a better way would be "value ... is unknown"

Reply:
Done!